packages/babel/src/traversal/scope/index.js

```diff
@@ -829,14 +829,15 @@ export default class Scope {
 
     var unique = opts.unique;
     var kind   = opts.kind || "var";
+    var blockHoist = opts._blockHoist || 2;
```

Comment:
This will need to be `opts._blockHoist == null ? 2 : opts._blockHoist` in case it's `0`.